TUTORIAL.md

```diff
@@ -15,15 +15,14 @@ All other methods require a valid JWT set in the header as a bearer token - `Aut
 ### userRequestAuthCode
 
 #### query
-```
+```shell
```

Comment:
the URI in this code block is outdated. Can we update to api.staging.galoy.io?

Reply:
Done!